heareval/score.py

```diff
@@ -76,7 +99,12 @@ def __init__(
             self.name = name
         self.maximize = maximize
 
-    def __call__(self, predictions: Any, targets: Any, **kwargs) -> float:
+    def __call__(self, *args, **kwargs) -> float:
```

Comment:
Wait so why is `return -> float` if in some cases it can be `Tuple[Tuple[str, float]]`? Where do `Tuple[str, float]` come from? (And if not's here, can you document in `validate_score_return_type` where the tuples might come from?

Reply:
Also please add a comment explaining why we have __call__ and compute, and why